.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "Earthly Docker",
+	"image": "mcr.microsoft.com/devcontainers/universal:2",
+	
+    // Installs earthly with docker and bootstraps it
+	"features": {
```

Comment:
It looks like this contains a mix of tabs & spaces - can you make that consistent?